script/publish_views

```diff
@@ -9,6 +9,8 @@ import os
 from google.cloud import bigquery
 import sqlparse
 
+VIEWS_TO_AUTHORIZE = ("sql/monitoring/payload_bytes_error_structured/view.sql",)
```

Comment:
Having `payload_bytes_error_telemetry` at the get-go would be nice too.

Reply:
That was an oversight. Adding all 4 of these new views to the list.